go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/ngaut/sqltop
 go 1.13
 
 require (
+	code.cloudfoundry.org/bytefmt v0.0.0-20190819182555-854d396b647c
```

Comment:
why we need this package?

Reply:
the previous PR introduced this library in stat.go for displaying human-readable IO flow status, but I didn't run `go mod tidy` then.